sci-libs/opencascade/opencascade-7.5.2.ebuild

```diff
@@ -0,0 +1,175 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+# TODO:
+# check the src files referenced in 51opencascade, i.e. resources and the like
+
+EAPI=7
+
+inherit cmake flag-o-matic
+
+MY_PV="$(ver_rs 1- '_')"
+PV_MAJ="$(ver_cut 1-2)"
+
+DESCRIPTION="Development platform for CAD/CAE, 3D surface/solid modeling and data exchange"
+HOMEPAGE="https://www.opencascade.com"
+SRC_URI="https://git.dev.opencascade.org/gitweb/?p=occt.git;a=snapshot;h=refs/tags/V${MY_PV};sf=tgz -> ${P}.tar.gz"
+S="${WORKDIR}/occt-V${MY_PV}"
+
+LICENSE="|| ( Open-CASCADE-LGPL-2.1-Exception-1.0 LGPL-2.1 )"
+SLOT="${PV_MAJ}"
+KEYWORDS="~amd64 ~arm64 ~x86"
+IUSE="debug doc examples ffmpeg freeimage gles2 json optimize tbb vtk"
+
+REQUIRED_USE="?? ( optimize tbb )"
+
+# There's no easy way to test. Testing needs a rather big environment
+# properly set up.
+RESTRICT="test"
+
+RDEPEND="
+	app-eselect/eselect-opencascade
+	dev-cpp/eigen
+	dev-lang/tcl:0=
+	dev-lang/tk:0=
```

Comment:
Why? Both packages have subslots, both are used at runtime (and build-time) by the package, so changes in slot's might need a rebuild of the package.

Reply:
Slot refer to the 0, not the := binding operator, keep :=

In both cases they typically don't get added until they're needed, binding operator is, slot isn't.

Reply:
I can remove them, if you care. I don't. I never add slot 0, so they are probably left-over from the time before I maintained the ebuild. I usually don't touch them, because they don't do any harm, and I really don't care about them, as long as I don't get any issues with them. And I see slot 0 used a lot, and I mean, A LOT in ebuilds.

Reply:
In a lot of case it's because it used to have slots, openssl for instance had slots in the past and reverted back to SLOT=0 and replaced these by the openssl-compat package.

However, that is getting old and not really needed on openssl anymore either.

Reply:
> **Edit:** Doesn't hurt to cleanup old cruft when doing bumps, much like old python compats that aren't recognized anymore even though they aren't "harmful".

Agreed